certified-connectors/Verified/apiProperties.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "username",
+          "description": "The username for this api",
+          "tooltip": "Provide the username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "password",
+          "description": "The password for this api",
+          "tooltip": "Provide the password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "companyid": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Company id is provided by verified. Can be found in the URL by logging in to app.verified.eu",
+          "displayName": "Company ID",
+          "tooltip": "Provided by verified, see description"
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@BenediktBergmann You agreed to not have this iconBrandColor for your connector.

![image](https://user-images.githubusercontent.com/56276394/155226859-a1605281-1333-489e-a941-5ab1da2f1734.png)

Reply:
Sorry, my fault. Should be changed now.